src/Test/IntegrationCase.php

```diff
@@ -128,13 +128,17 @@ public function getRequirement($name)
         if (!is_string($name)) {
             throw new \InvalidArgumentException(sprintf(
                 'Requirement key must be a string, got "%s".',
-                is_object($name) ? get_class($name) : gettype($name).'#'.$name));
+                is_object($name) ? get_class($name) : gettype($name).'#'.$name
+            ));
         }
 
         if (!array_key_exists($name, $this->requirements)) {
-            throw new \InvalidArgumentException(sprintf(
+            throw new \InvalidArgumentException(
+                sprintf(
                 'Unknown requirement key "%s", expected any of "%s".',
-                $name, implode('","', array_keys($this->requirements)))
+                $name,
+                implode('","', array_keys($this->requirements))
+            )
```

Comment:
The indention is wrong, but I think we established it wasn't the responsability of this fixer. And it looks like there is no fixer that can fix it afterwards.